### PR TITLE
applications: nrf_desktop: Update config channel doc

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds module variant to the config channel documentation.

### CODE REVIEW

**Review by greg-fer — APPROVED** [review]
There were some formatting issues related to missing spaces, which were breaking the structure of the pages. Fixed now.

File: scripts/hid_configurator/README.rst [hunk 1]
```diff
@@ -133,7 +133,7 @@ Configuring device runtime options
 ==================================
 
 The script can pass the configuration values to the linked FW module using the ``config`` command.
-Use the following syntax to display list of configurable modules:
+Use the following syntax to display list of modules that can have device runtime options configured:
```
Comment (MarekPieta) [inline scripts/hid_configurator/README.rst:136]: Can we leave it as `configurable modules`? I think it's simpler and the configured options actually belong to an application module that is part of the firmware, not to a whole device

Comment (MarekPieta) [reply scripts/hid_configurator/README.rst:136]: Also `ble_bond` is a `configurable module`, but it does not actually configure the device runtime options (you can trigger Bluetooth scanning or peer erase using `config` command for this module)

Comment (greg-fer) [reply scripts/hid_configurator/README.rst:136]: OK, that's fine.

**Review by pdunaj — APPROVED** [review]

**Comment by greg-fer** [comment]
@pdunaj , looks like you forgot to squash my commit...

**Comment by pdunaj** [comment]
Hi @greg-fer , yes. I missed it :( I am not sure we will be able to do anything about it as fixing would require history rewrite. Besides the strange message the commit is fine so I don't think it's worth it (@carlescufi what do you think?)

@greg-fer I will double check the commit messages but I have one favor to ask. Could you not include signoff in review commit messages? This would help to avoid similar issue in the future as CI will reject the commit and merge won't be possible.

Once again sorry <bowing>

**Comment by carlescufi** [comment]
> Hi @greg-fer , yes. I missed it :( I am not sure we will be able to do anything about it as fixing would require history rewrite. Besides the strange message the commit is fine so I don't think it's worth it (@carlescufi what do you think?)

In this particular case this is not a problem, the commit was fairly clean. I will probably work on a GitHub Action soon that will deal with this and other similar issues.

**Pull request merged**